swift/apple/FirezoneKit/Sources/FirezoneKit/Features/SettingsView.swift

```diff
@@ -176,95 +164,50 @@ public struct SettingsView: View {
         .navigationTitle("Settings")
         .navigationBarTitleDisplayMode(.inline)
       }
+      .alert(
+        "Saving settings will sign you out",
+        isPresented: $isShowingConfirmationAlert,
+        presenting: confirmationAlertContinueAction,
+        actions: { confirmationAlertContinueAction in
+          Button("Continue") {
+            confirmationAlertContinueAction.performAction(on: self)
+          }
+          Button("Cancel", role: .cancel) {
+            // Nothing to do
+          }
```

Comment:
Does order matter here?

```suggestion
          Button("Cancel", role: .cancel) {
            // Nothing to do
          }
          Button("Continue") {
            confirmationAlertContinueAction.performAction(on: self)
          }
```

Reply:
It doesn't. SwiftUI will take care of placing the cancel-role-button to the left. Nevertheless, I agree it's a good idea to change the order in the code anyway.

Reply:
Fixed in commit c9d6145.